charts/kyverno/values.yaml

```diff
@@ -493,6 +493,11 @@ webhooksCleanup:
 # -- A writable volume to use for the TUF root initialization.
 tufRootMountPath: /.sigstore
 
+# -- If set the volume for tufRootMountPath is freely configurable below "- name: sigstore". If no value is set an emptyDir is used.
+sigstoreVolume: {}
```

Comment:
don't we want `emptyDir: {}` to be the default value ?

Reply:
yes, it's done in the template currently.
i can also set the default here.